### PR TITLE
document changes from PR#3037

### PR DESCRIPTION
**Pull request by martgras**
## Description:

Document / Clarify changes from https://github.com/esphome/esphome/pull/3037

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#3037

## Checklist:

  - [ ] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/sensor/tcs34725.rst [hunk 1]
```diff
@@ -49,18 +49,13 @@ required to be set up in your configuration for this sensor to work.
...
-  All options from :ref:`Sensor <config-sensor>`.
-- **blue_channel** (*Optional*): Get the percentage of how strongly the blue color channel is activated.
-  All options from :ref:`Sensor <config-sensor>`.
-- **clear_channel** (*Optional*): Get the percentage of how strongly the clear (without a color filter)
-  channel is activated. All options from :ref:`Sensor <config-sensor>`.
+- **red_channel** (*Optional*): Get the percentage of red color channel in relation to the ``clear_channel``.
+- **green_channel** (*Optional*): Get the percentage the green color channel in relation to the ``clear_channel``.
+- **blue_channel** (*Optional*): Get the percentage of the blue color channel in relation to the ``clear_channel``.
+- **clear_channel** (*Optional*): Get the percentage of the clear (without a color filter)
+  channel in relation to the maximum value for the choosen ``integration_time``.
```
Comment (oxan) [inline components/sensor/tcs34725.rst]: Let's leave the sensor references in here and clarify that it's a relative value:
```suggestion
- **red_channel** (*Optional*): Value of the red color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.
- **green_channel** (*Optional*): Value of the green color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.
- **blue_channel** (*Optional*): Value of the blue color channel relative to the clear channel, as a percentage.
  All options from :ref:`Sensor <config-sensor>`.
- **clear_channel** (*Optional*): Vaule of the clear (without a color filter) channel, relative to the maximum value for
  the chosen integration time. All options from :ref:`Sensor <config-sensor>`.
```

File: components/sensor/tcs34725.rst [hunk 2]
```diff
@@ -49,18 +49,13 @@ required to be set up in your configuration for this sensor to work.
...
-  All options from :ref:`Sensor <config-sensor>`.
-- **clear_channel** (*Optional*): Get the percentage of how strongly the clear (without a color filter)
-  channel is activated. All options from :ref:`Sensor <config-sensor>`.
+- **red_channel** (*Optional*): Get the percentage of red color channel in relation to the ``clear_channel``.
+- **green_channel** (*Optional*): Get the percentage the green color channel in relation to the ``clear_channel``.
+- **blue_channel** (*Optional*): Get the percentage of the blue color channel in relation to the ``clear_channel``.
+- **clear_channel** (*Optional*): Get the percentage of the clear (without a color filter)
+  channel in relation to the maximum value for the choosen ``integration_time``. 
 - **illuminance** (*Optional*): Get the total illuminance of the sensor in lx.
-  All options from :ref:`Sensor <config-sensor>`.
```
Comment (oxan) [inline components/sensor/tcs34725.rst:61]: I wouldn't remove this, now it's unclear what you should specify for this setting.

File: components/sensor/tcs34725.rst [hunk 3]
```diff
@@ -49,18 +49,13 @@ required to be set up in your configuration for this sensor to work.
...
 - **color_temperature** (*Optional*): Get the calculated color temperature of the light in Kelvin.
-  All options from :ref:`Sensor <config-sensor>`.
```
Comment (oxan) [inline components/sensor/tcs34725.rst:63]: Same.

File: components/sensor/tcs34725.rst [hunk 4]
```diff
@@ -72,6 +67,8 @@ Configuration variables:
 - **update_interval** (*Optional*, :ref:`config-time`): The interval to check the
   sensor. Defaults to ``60s``.
 
+Since the raw R+G+B values reported by the sensor are approximately equal to the clear value, ``red_channel``, ``green_channel`` and ``blue_channel`` percentages are calculated in relation to the ``clear_channel``.
```
Comment (oxan) [inline components/sensor/tcs34725.rst]: Isn't this already explained above?

**Review by oxan — CHANGES_REQUESTED** [review]

File: components/sensor/tcs34725.rst (hunk 2 above)
Comment (martgras) [reply components/sensor/tcs34725.rst:61]: ok

File: components/sensor/tcs34725.rst (hunk 4 above)
Comment (martgras) [reply components/sensor/tcs34725.rst]: removed

**Review by oxan — APPROVED** [review]

File: components/sensor/tcs34725.rst [hunk 5]
```diff
@@ -49,18 +49,16 @@ required to be set up in your configuration for this sensor to work.
...
   All options from :ref:`Sensor <config-sensor>`.
-- **green_channel** (*Optional*): Get the percentage of how strongly the green color channel is activated.
+- **green_channel** (*Optional*): Value of the green color channel relative to the clear channel, as a percentage.
   All options from :ref:`Sensor <config-sensor>`.
-- **blue_channel** (*Optional*): Get the percentage of how strongly the blue color channel is activated.
+- **blue_channel** (*Optional*): Value of the blue color channel relative to the clear channel, as a percentage.
   All options from :ref:`Sensor <config-sensor>`.
-- **clear_channel** (*Optional*): Get the percentage of how strongly the clear (without a color filter)
-  channel is activated. All options from :ref:`Sensor <config-sensor>`.
+- **clear_channel** (*Optional*): Vaule of the clear (without a color filter) channel, relative to the maximum value for
```
Comment (jesserockz) [inline components/sensor/tcs34725.rst]:
```suggestion
- **clear_channel** (*Optional*): Value of the clear (without a color filter) channel, relative to the maximum value for
```

**Review by jesserockz — APPROVED** [review]

**Pull request merged**